lib/sys_test.py

```diff
@@ -86,13 +86,17 @@ def TestExitInvalidArgs():
     assert False
 
 
-def test_getframe(self):
-  self.assertRaises(TypeError, sys._getframe, 42, 42)
-  self.assertRaises(ValueError, sys._getframe, 2000000000)
-  self.assertTrue(
-    SysModuleTest.test_getframe.im_func.func_code \
-    is sys._getframe().f_code
-  )
+def TestGetFrame():
+  try:
+    sys._getframe(42, 42)
+  except TypeError:
+    assert True
```

Comment:
I would make this a pass statement and put an else block with assert False so that if no exception is raised the assertion triggers. Ditto below.